content/en/docs/concepts/what-is-istio/index.md

```diff
@@ -74,18 +74,6 @@ secure {{<gloss>}}pod{{</gloss>}}-to-pod or service-to-service communication at
 
 Refer to the [Security concepts guide](/docs/concepts/security/) for more details.
 
-### Policies
```

Comment:
Shoudn't we leave a high level Policies section on this page? It makes sense to remove the policy adapters reference and the link to more details, but even though Mixer is deprecated, there are other ways to configure policies and enforce rules in Istio, so it seems to me that we should at least leave some high level overview in the what-is-istio page about policies.

Reply:
I think we should just remove it from the concept page if it's deprecated, the concept page should always be consistent with the current istio version. Users should refer to the older docs for more information about the mixer v1.

And I think we can add to the upgrade note to make it clear that the policy (which is equivalent to mixer v1) is deprecated and the alternative options as mentioned by @mandarjog

Reply:
I think the problem I see stems from the fact that you guys are equating the 4th piller with Mixer, which is true at the implementation level, but at the concepts (what is istio) level, Mixer is an implementation detail. If we completely remove Policies from the overview, and go with the three pillar doc, we are saying that Istio only provides 3 kinds of functionality. Users can:

1. Manage the traffic
2. Secure their services
3. Observe their services

This list is leaving out some functionality, like setting rate limits, that Istio used to provide. Since it can still be done (just not using Mixer now), I think we should still keep the 4th pillar, or as I suggested below, add Policies to the Security pillar. I don't think that completely removing any mention of policies in the overview doc and expecting people to find out about policies by digging into the details of deprecated docs and upgrade notes is a very nice UX.

Anyway, that's just my opinion. Maybe others feel a functionality regression at the high level overview isn't a problem.

Reply:
If we remove the Policies section, how about changing the Security section to include the bullets about policy. Change the `### Security` section to this:
 ```
### Security and Policies

Istio’s security capabilities free developers to focus on security at the application level. Istio provides the underlying secure communication channel, and
manages authentication, authorization, and encryption of service communication at scale. With Istio, service communications are secured by default,
letting you enforce policies consistently across diverse protocols and runtimes -- all with little or no application changes.

While Istio is platform independent, using it with Kubernetes (or infrastructure) network policies, the benefits are even greater, including the ability to
secure {{<gloss>}}pod{{</gloss>}}-to-pod or service-to-service communication at the network and application layers.

Istio also lets you configure custom policies for your application to enforce rules at runtime such as:

* Rate limiting to dynamically limit the traffic to a service
* Denials, whitelists, and blacklists, to restrict access to services
* Header rewrites and redirects

Refer to the [Security concepts guide](/docs/concepts/security/) for more details.
```

Reply:
> I think the problem I see stems from the fact that you guys are equating the 4th piller with Mixer, which is true at the implementation level, but at the concepts (what is istio) level, Mixer is an implementation detail.

Without Mixer, I don't feel we even need the `policy` concept at all, the `policy` concept exists because we have some dedicated Mixer API to use it. With the deprecation of Mixer, the Mixer API is gone and we do not want anyone to use it anymore.

> This list is leaving out some functionality, like setting rate limits, that Istio used to provide. Since it can still be done (just not using Mixer now), I think we should still keep the 4th pillar,

I understand it is a functionality regression but this is what happening anyway in Istio 1.5, and the doc should reflect this change clearly.

On the other hand, yes, the feature (rate limiting, control header, etc.) could be done but that is out of the scope of `policy` concept. You have to use the `EnvoyFilter` to configure Envoy directly which is quite hard and breaking class change. We do not have any documentation on this at all.

Even if we have the documentation, it should really go to the `Manage the traffic`  section because it has nothing to do with the `Policy` concept as it's all about the configuring the Envoy API directly.


@louiscryan @smawson @linsun @rshriram, I'm wondering what does TOC think here? Should we remove the `policy` from the concept or should we keep it but merge it with other sections? Thanks.

Reply:
> Even if we have the documentation, it should really go to the Manage the traffic section

@yangminzhu That's a good point. Maybe the right way to document it is to remove the `### Policies` section, as you've done, but then add these 2 features to the `### Traffic Management` section:
* Rate limiting to dynamically limit the traffic to a service
* Header rewrites and redirects

and this one to the `### Security` section:
* Denials, whitelists, and blacklists, to restrict access to services

This way, the "what is Istio" doc still includes all the functionality that it previously had.

Reply:
For `Denials, whitelists, and blacklists, to restrict access to services`, I can add a task under security but probably with a different name because the term "denials", "whitelists" and "blacklists" doesn't exist in authorization policy.

For `Rate limiting to dynamically limit the traffic to a service` and `Header rewrites and redirects`, I don't know who would will own and work on it, is it the Network WG or the Policies & Telemetry WG? @mandarjog @duderino 

Beyond this, @frankbu do you think we can get this PR merged first? We probably want to split the task changes in a follow-up PR and follow the normal process by creating and reviewing in a google doc before sending out a PR.

Reply:
@yangminzhu Sounds good. We can do the task changes in a follow-up PR.